logging/testing.go

```diff
@@ -26,17 +27,20 @@ func TestLogger(tb testing.TB) *slog.Logger {
 	tb.Helper()
 
 	w := &testingWriter{tb}
-	return slog.New(slog.NewTextHandler(w, &slog.HandlerOptions{
+	level := slog.Level(math.MinInt)
```

Comment:
Could you add a comment here? This reasoning behind this isn't immediately obvious to me.

Reply:
Done.